### DISCUSSION TITLE
Does Portman provide support for the Openapi AMF framework extension

### DISCUSSION THREAD

**Discussion by PaulCluver** [apideck-libraries/portman#579] [Q&A] [answered]
Hi there, 

I am using Portman to try and generate postman collections for an API experience layer that is controlled by Mulesoft. The request body is not being generated with the expected sample data. 

Will Portman support the inclusion of the `x-amf-union` tag? 

Kind regards,
Paul

**Comment by thim81** [comment]
The conversion uses the mostly standard OpenApi properties to map them to Postman properties. x-amf-union seems like a very specific custom openAPI extension. It will not be converted. 

What is the content of the x-amf-union property? Depending on the purpose You could convert it with a custom script into a default openAPI example?

**Reply by PaulCluver** [reply]
Hi Tim, 

The content of the x-amf-union property is simply an array of $refs. 

When you say, "...convert it with a custom script into a default openAPI example?", would you envision this happening outside of Portman? In other words: 

1. Get the API Spec
2. Convert API Spec into default openAPI example
3. Run portman

Thanks in advance,
Paul.

**Reply by thim81** [reply, marked as answer]
That would be the scenario. 

It could be a bash script or custom JavaScript that you run before the Portman conversion.

**Reply by PaulCluver** [reply]
Thanks so much Tim
